src/providers/twitch/TwitchChannel.cpp

```diff
@@ -621,7 +592,9 @@ const QString &TwitchChannel::popoutPlayerUrl()
 
 int TwitchChannel::chatterCount()
 {
-    return this->chatterCount_;
+    return this->hasModRights()
+        ? this->chatterCount_
+        : this->accessChatters()->getNumChatters();
```

Comment:
These two numbers are very different and used for different purposes, portraying them both as the same number is in my opinion confusing.

Reply:
`chatterCount_` has been removed because it cannot be used by non mods anyways. I've simplified this function down to just using `this->accessChatters()->getNumChatters()`